src/handlers/comment/handlers/query.ts

```diff
@@ -25,9 +25,15 @@ export const query = async (body: string) => {
   const repo = payload.repository;
 
   if (user) {
-    const data = await getAllAccessLevels(user, repo.full_name);
+    let data = await getAllAccessLevels(user, repo.full_name);
     if (!data) {
-      return `Error retrieving access for @${user}`;
+      logger.info(`Access info does not exist for @${user}`);
+      data = {
+        multiplier: false,
+        priority: false,
+        time: false,
```

Comment:
I feel that it might make sense to default `time` to true. Can that be handled by this or is this a separate bounty?

I think contributors should be able to add `time` estimates to the work, and project managers to set `priority`

Reply:
fixed.

Reply:
What happens when I set your permission for something unrelated? Wouldn't the database initialize `time` as `false` too?

Reply:
YEah, the bot need to upsert the initialized info to the table at this line.
'll add that feature on this PR after the weekend. @pavlovcik

Reply:
Faster to do as a new issue.